frontend_lib/src/component/Timeline/AddFileToCommentButton.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'react-i18next'
+import PopupUploadFile from '../../container/PopupUploadFile.jsx'
+
+// const MAXIMUM_IMAGE_SIZE = 10 * 1024 * 1024 // 10mo
+// const ALLOWED_IMAGE_MIMETYPES = [
+//   'image/jpeg',
+//   'image/png',
+//   'image/bmp',
+//   'image/gif',
+//   'image/webp'
+// ]
+
+export class AddFileToCommentButton extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      showUploadPopup: false
+    }
+  }
+
+  handleDisplayPopupUpload = e => {
+    e.preventDefault()
+    this.setState(prev => ({ showUploadPopup: !prev.showUploadPopup }))
+  }
+
+  handleClosePopup = () => {
+    this.setState({ showUploadPopup: false })
+  }
+
+  handleValidatePopupOverride = fileToUploadList => {
+    this.handleClosePopup()
+    this.props.onValidateCommentFileToUpload(fileToUploadList)
+  }
+
+  render () {
+    const { props, state } = this
+
+    return (
+      <>
+        <button
+          className='AddFileToCommentButton'
+          onClick={this.handleDisplayPopupUpload}
+          style={{ color: props.color }}
+        >
+          <i className='fas fa-paperclip' />
+        </button>
```

Comment:
Can't you use `<IconButton/>` or `<button> <Icon/> </button>` here?

Reply:
will fix

Reply:
fixed